constants/responses.js

```diff
@@ -2,15 +2,64 @@
 
 module.exports = {
   FR: {
-    noIntent: "Je n'ai pas bien compris ta demande, pour voir la liste des questions auxquelles je peux répondre consulte ce site https://www.ovh.com/manager/web/#/labs/chatbot",
+    connectedAs: "Tu es connecté en tant que %s :ok_hand:",
+    domainEditDns: "Tu dois modifier la zone DNS de ton domaine afin d'y ajouter le champs A avec l'ip de ton hébergement web, cette ip se trouve sur la page d'informations de ton hébergement dans l'espace client OVH.",
+    disconnected: "Tu n'es pas correctement connecté à ton compte OVH :( , il te suffit de me demander 'connecte moi' pour te reconnecter.",
+    dnsEditDns: "Tu peux modifier tes serveurs via l'espace client OVH",
+    feedbackBadUnderstanding: "Mauvaise compréhension",
+    feedbackHelp: "Est-ce que cette réponse t'as aidé ?",
+    feedbackNo: "Non",
+    feedbackThanks: "Merci pour ton avis :+1:",
+    feedbackYes: "Oui",
+    goToManager: "Accéder à l'espace client :point_right:",
+    hostingSelectSite: "Sélectionne le site concerné (%1$s/%2$s)",
+    hostingSelectHost: "Sélectionne l'hébergement web sur lequel est installé ton site (%1$s/%2$s)",
+    hostingNoSite: ":globe_with_meridians: Tu n'as pas d'hébergement web :(",
+    hostingWrongSite: "Tu as du sélectionner le mauvais hébergement web associé à ce domaine. :thinking_face:",
```

Comment:
Tu as dû

Or maybe you can say it this way :
Il semble que l'hébergement web que tu as sélectionné n'est pas celui associé à ce domaine.